ponyo/utils.py

```diff
@@ -56,35 +55,7 @@ def setup_dir(config_filename):
             print("creating new directory: {}".format(each_dir))
             os.makedirs(each_dir, exist_ok=True)
 
-        # Check if NN architecture directory exist otherwise create
```

Comment:
do you still need to check if `training_stats_dir`, `vae_model_dir`, `local_dir`, and `simulated_data_dir` exist?

Reply:
Yes I should add back in a test for this. Thank you for pointing this out

Reply:
Actually these lines are still checking if the directories exist and if not creating them: https://github.com/ajlee21/ponyo/blob/a38745cb72ff4c289a398e662327c45d31c683f9/ponyo/utils.py#L53